recipes/compss/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "pycompss" %}
+{% set version = "2.4" %}
+{% set sha256 = "237a8ed3842a6d836a330f4c707cd12781ef6cfad58668d52a84be286384a0ba" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  #url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  #url: http://compss.bsc.es/repo/sc/stable/COMPSs_{{ version }}.tar.gz
+  url: http://bscgrid05.bsc.es/~lezzi/COMPSs_{{ version }}.tar.gz
```

This doesn't look like an official source? Would this be better:
```suggestion
  url: https://github.com/bsc-wdc/compss/archive/{{ version }}.tar.gz
```
Also can you remove the commented lines above.

---

No, for the moment is better to leave this url

---

Why?

---

Because the official release contains extensions that cannot be compiled on all the systems and we don't have yet a parametric installation script.

---

In that case it sounds like this [should use patches](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selectors#patches) on top of the upstream tarball.